### PR TITLE
Add multiple subtitles support

### PR DESCRIPTION
**Pull request by grunjol**
No subtitles
image

add new with click if no subtitles available

Subtitles available
image

once a subtitle is available, add more subtitles with Ctrl+[CC] click

Subtitles enabled and selected
image

on [CC] click, show available subtitles

Subtitles menu
image

* use [languagedetect](https://www.npmjs.com/package/languagedetect) (no deps) to detect subtitle language

### CODE REVIEW

**Comment by grunjol** [comment]
Added a [couple](https://github.com/mafintosh/srt-to-vtt/issues/3) of [issues](https://github.com/mafintosh/srt-to-vtt/issues/4) with srt-to-vtt to handle CP1252 encoded .srt files

**Comment by rom1504** [comment]
nice

File: renderer/index.js [hunk 1]
```diff
@@ -556,17 +563,43 @@ function addSubtitle (file) {
     // Set the cue text position so it appears above the player controls.
     // The only way to change cue text position is by modifying the VTT. It is not
     // possible via CSS.
+    var langDetected = (new LanguageDetect()).detect(buf.toString().replace(/(.*-->.*)/g, ''), 2)
+    console.log(langDetected)
```
Comment (feross) [inline renderer/index.js]: remove `console.log`

File: renderer/index.js [hunk 2]
```diff
@@ -556,17 +563,43 @@ function addSubtitle (file) {
...
     var track = {
       buffer: 'data:text/vtt;base64,' + subtitles.toString('base64'),
-      language: 'Language ' + state.playing.subtitles.tracks.length,
+      label: langDetected,
       selected: true
     }
+    state.playing.subtitles.tracks.forEach(function (trackItem) {
+      trackItem.selected = false
+      if (trackItem.label === track.label) {
+        track.label = isNaN(track.label.slice(-1))
```
Comment (feross) [inline renderer/index.js]: `Number.isNaN` is preferable to `isNaN`

**Comment by grunjol** [comment]
@feross changes done

**Comment by grunjol** [comment]
rebased

**Comment by feross** [comment]
I'll take a final look in one second

**Comment by feross** [comment]
LGTm

**Pull request merged**